### DISCUSSION TITLE
Reorder Tabs

### DISCUSSION THREAD

**Discussion by andig** [community/community#2925] [Discussions]
This is the current layout:

<img width="849" alt="Screenshot 2020-12-10 at 12 12 51" src="https://user-images.githubusercontent.com/184815/101765078-1c31a400-3ae1-11eb-8a53-a83e98fb9eae.png">

Right now is discussions squeezed between coding topics.

I would suggest to move the Discussions tab either:
- left next to issues (as they are similar)
- right next to wiki (as wiki is the first non-code topic)

**Comment by becca** [comment]
Thank you for this feedback. Early on we tried a few different locations for it but overall the community wanted it to be an organic workflow moving from Work items (Issues), Code (PRs) to Discussions as it matched the way they prioritize events on GitHub. 

Would having Discussions before Issues not break your workflow? Many folks early on kept clicking Discussions in that scenario and found they really meant to be in Issues.

**Comment by andig** [comment]
> Would having Discussions before Issues not break your workflow? Many folks early on kept clicking Discussions in that scenario and found they really meant to be in Issues.

Fair question. It feels very unnatural between PRs and Actions though as my typical flow is from PR into build status. So I'd vote for either _left_ of issues or _right_ of projects. Left of issues looks unnatural since "Home" is actually "Code". So for time being maybe next to Wiki as more community-oriented feature?

**Comment by Andre601** [comment]
I personally see the current order just fine, as I can imagine it like this:

User creates an issue in the issues tab.
Maintainer creates a PR to adress the issue created by the User.
Either user or Maintainer start a discussion to look into how the PR can be improved more.

In that scenario would it be "issues -> Pull requests -> Discussions", so just how it is ordered right now.
In short does it imo represent the current flow of coding.

**Comment by andig** [comment]
> Either user or Maintainer start a discussion to look into how the PR can be improved more.

Your milage may vary, this is at least not my flow. PR is already liked to issue and discussion happens in either of the two (typically the issue). A separate this discussion thread for this purpose is imho not useful.

> In short does it imo represent the current flow of coding.

I don't feel it does as the current flow works entirely without discussions. For my projects, all discussions are either in a 3rd party forum (like with many OS organizations) or on mailing list. The use case I see is replacing the need for those inside GH.

The discussions feature it more relevant for broader discussions ("how can I") that don't fit immediately the issue/feature request pattern.
